di_website/templates/includes/forms/publication-filter.html

```diff
@@ -1,5 +1,29 @@
-<form method="get" action="{{self.pageurl}}" class="form">
-  <div class="form-field form-field--inline-four">
+<form method="get" action="{{self.pageurl}}" class="form resources-filters
+resources-filters
+resources-filters
+resources-filters
+resources-filters">
```

Comment:
Why??

Reply:
Get in the habit of reviewing the changes in every file before you commit it. Should at least help u avoid these kinds of strange things.

Reply:
Downside of using `ctrl + d` in vscode without checking the results; Its now updated